docker-compose-infra.yml

```diff
@@ -6,7 +6,7 @@ services:
     volumes:
       - psql_db:/var/lib/postgresql/data
     ports:
-      - 5432:5432
+      - 5434:5432
```

Comment:
Do we need this port change? we will have to change the env var `POSTGRES_CONNECTION_URL` value to have 5434